products/early_access_features/frontend/earlyAccessFeatureLogic.ts

```diff
@@ -164,7 +164,6 @@ export const earlyAccessFeatureLogic = kea<earlyAccessFeatureLogicType>([
         ],
     }),
     selectors({
-        mode: [(_, p) => [p.id], (id): 'view' | 'edit' => (id === 'new' ? 'edit' : 'view')],
```

Comment:
I copied from this logic, and noticed this wasn't used, so I removed it from here too